orchagent/fdborch.cpp

```diff
@@ -157,6 +179,21 @@ void FdbOrch::doTask(Consumer& consumer)
     }
 }
 
+void FdbOrch::updateVlanMember(const VlanMemberUpdate& update)
+{
+    SWSS_LOG_ENTER();
+
+    if (!update.add) return; // we need only additions
+
+    string port_name = update.member.m_alias;
+    auto fdb_list = std::move(saved_fdb_entries[port_name]);
```

Comment:
Just thinking, lets say port 1 is member of Vlan 1 and Vlan 2 and fdb_entries are saved per port. Now when` port 1` is added to` Vlan 1`, is it the case that fdb entries on` Vlan 2`, `port 1` is also removed and added even-though we still didn't get a member add of `port 1` to `Vlan 2`?

Reply:
Valid point. Thanks Prince. I'll add a fix ASAP.

Reply:
So I've double checked this case.
I programmed following logic here. As soon as we get vlan update we try to reinsert fdb entries on the port. If something wrong with some entry, it will be reinserted back to the hash. I'll add comments describing the logic.